reviewer/gh.go

```diff
@@ -61,14 +64,20 @@ func getCommentSuccessScore(comment string) int {
 	return score
 }
 
+// GetPullRequestInfos returns the list of pull requests and their success score based on comments
 func GetPullRequestInfos(client *github.Client, owner string, repo string) (*PullRequestInfoList, error) {
+	//TODO: At this moment if there's a lot of PR, does not returns the full list, needs pagination.
+	//      Also maybe we need to take care about how much requests are done in order to not
+	//        exceed the cuota
```

Comment:
We might use [PullRequestListOptions|https://godoc.org/github.com/google/go-github/github#PullRequestListOptions] to filter the received PRs received by status = open.
About the quota, do you mean the [rate limit|https://developer.github.com/v3/rate_limit/]? In that case, supposedly, go-github [implements caching|https://godoc.org/github.com/google/go-github/github#hdr-Conditional_Requests]. Also, I would propose to contribute some throttling mechanism to go-github, in the future.